<launch>

  <!--读bag文件-->
  <node pkg="rosbag" type="play" name="player" args="$(find record_data)/bagfiles/191112_f2.bag" respawn="true" />    

  <!--自车Marker-->
  <node pkg="lidar_grid" type="ego_car_node" name="ego_car" />

  <!--激光雷达目标点云聚类-->
  <node pkg="detection" type="lidar_cluster_node" name="lidar_cluster_node" output="screen">
    <rosparam command="load" file="$(find detection)/param/lidar_cluster.yaml" />
  </node>

  <!--接收MPC单片机CAN消息-->
  <node pkg="raw_data" type="MPC_raw_node" name="MPC_raw_node" output="screen">
    <rosparam command="load" file="$(find raw_data)/param/front_raw.yaml" />
  </node>

  <!--目标跟踪模块-->
  <node pkg="detection" type="tracker_node" name="tracker_node" output="screen">
    <rosparam command="load" file="$(find detection)/param/front_tracker.yaml" />
  </node>

  <!--rviz可视化-->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find detection)/rviz/detection.rviz" required="true"/>   

  <!--rqt_plot-->
  <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot" />

</launch>